src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -175,10 +171,10 @@
  * </p>
  *
  * <pre>
- * class Foo {
- *   int field1; // ok
- *   double field2; // ok
- *   long field3, field4 = 10L, field5; // ok
+ * class FirstClass {
+ *   int var1; // ok
+ *   double var2; // ok
+ *   long var3, var4 = 10L, var5; // ok
  * }
```

Comment:
All java code blocks should be same.
Difference is only in trailing comments, as violation location is changing as we change property values in config.

Reply:
This gave a clearer picture. Have made modifications accordingly.

Reply:
Fixing CI failure